src/components/_shared/styled-button.js

```diff
@@ -0,0 +1,71 @@
+import styled from '@emotion/styled';
+import Arrow from '../../assets/arrow.svg';
+import React from 'react';
+
+export const StyledButton = styled.button`
+  font-weight: 800;
+  margin: 10px 0;
+  white-space: nowrap;
+  position: relative;
+  border: none;
+  padding: 6px 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:before {
+    border: 1px solid #fff;
```

Comment:
there should be a css variable for the white color use it instead.
I feel like we could add a snippet in shared for the before block, so we can reuse it in the image container and here